bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Cursor.java

```diff
@@ -407,7 +390,7 @@ private void destroyHandle () {
 		handle.destroy();
 	}
 	zoomLevelToHandle.clear();
-	handle = 0;
+	this.isDestroyed = true;
```

Comment:
This `destroyHandle()` method could be combined with the `destroy()` method and all the comment/commented-out code could be removed from that method.